### ISSUE TITLE
nextcloud server uses too much RAM

### ISSUE THREAD

**Issue by razorfall** [nextcloud/serverinfo#56]
Heyy guys,

First of all, English is not my native language =)

I hope you can help me with my issue!
### Steps to reproduce

I don't know how to reproduce this issue...
### Expected behaviour

My nextcloud server shouldn't use 4 GB of RAM.
### Actual behaviour

My nextcloud server should use less then 2 GB of RAM with two users, but my server actually uses 4 GB RAM.
### Server configuration

**Operating system**:
Ubuntu 16.04

**Web server:**
Apache 2.4

**Database:**
MySQL

**PHP version:**
7.0

**Server version:** (see your admin page)
Nextcloud 10.0.1 (stable)

**Calendar version:** (see the apps page)
1.4.0

**Updated from an older installed version or fresh install:**
Fresh install

**Signing status (ownCloud/Nextcloud 9.0 and above):**

```
No errors have been found.
```

**List of activated apps:**

```
USER@HOST:~$ sudo -u www-data php occ app:list
Could not open input file: occ

```

Manually copied:

```
Activity - 2.3.2
Auditing / Logging - 1.0.0
Collaborative tags - 1.0.2
Comments - 1.0.0
Default encryption module - 1.3.1
Deleted files - 1.0.0
Federation - 1.0.1
File sharing - 1.0.0
First run wizard - 1.1
Gallery - 15.0.0
Notifications - 0.3.0
Password policy - 1.0.0
PDF viewer - 0.8.1
Provisioning API - 1.0.0
Server info - 1.1.1
Text editor - 2.1
Theming - 1.0.1
Update notification - 1.0.1
Usage report - 0.1.5
Versions - 1.3.0
Video player - 0.9.8
Bookmarks - 0.9.0
Calendar - 1.4.0
Contacts - 1.4.0.0
Tasks - 0.9.3
```

**The content of config/config.php:**

```
<?php
$CONFIG = array (
  'instanceid' => 'PRIVATE',
  'passwordsalt' => 'PRIVATE',
  'secret' => 'PRIVATE',
  'trusted_domains' => 
  array (
    0 => 'MYDOMAIN',
  ),
  'datadirectory' => '/home/CLOUDFOLDER/data',
  'overwrite.cli.url' => 'https://MYDOMAIN/NEXTCLOUD',
  'dbtype' => 'mysql',
  'version' => '9.1.1.5',
  'dbname' => 'CLOUDDATABASE',
  'dbhost' => 'localhost',
  'dbport' => '',
  'dbtableprefix' => 'DATABASEPREFIX',
  'dbuser' => 'DATABASEUSER',
  'dbpassword' => 'PRIVATE',
  'logtimezone' => 'UTC',
  'installed' => true,
  'memcache.local' => '\\OC\\Memcache\\APCu',
  'memcache.locking' => '\\OC\\Memcache\\Redis',
  'memcache.distributed' => '\\OC\\Memcache\\Redis',
  'filelocking.enabled' => 'true',
  'redis' => 
  array (
    'host' => 'localhost',
    'port' => 6379,
    'timeout' => 0.0,
  ),
  'loglevel' => 3,
  'versions_retention_obligation' => 'auto',
  'appcodechecker' => true,
  'updatechecker' => true,
  'updater.server.url' => 'https://updates.nextcloud.com/updater_server/',
);
```

**Are you using external storage, if yes which one:**
/home/Cloudfolder/data

**Are you using encryption:** yes/no
Yes

**Are you using an external user-backend, if yes which one:**
Webdav
### Client configuration

**Browser:**
Firefox
Android Browser

**Operating system:**
Win7 64Bit
Android 5 and 6

**CalDAV-clients:**
DavDroid 1.3.3-gplay
#### Web server error log

```
Error   PHP Undefined index: MemAvailable at /var/www/html/nextcloud/apps/serverinfo/lib/SystemStatistics.php#85    2016-10-14T05:01:04+00:00   USER
Error   PHP Undefined index: MemAvailable at /var/www/html/nextcloud/apps/serverinfo/lib/SystemStatistics.php#85    2016-10-14T05:01:04+00:00   USER
Error   PHP Undefined index: MemAvailable at /var/www/html/nextcloud/apps/serverinfo/lib/SystemStatistics.php#85    2016-10-14T05:01:02+00:00   USER
Error   PHP Undefined index: MemAvailable at /var/www/html/nextcloud/apps/serverinfo/lib/SystemStatistics.php#85    2016-10-14T05:00:54+00:00   USER
Error   PHP Undefined index: MemAvailable at /var/www/html/nextcloud/apps/serverinfo/lib/SystemStatistics.php#85    2016-10-14T05:00:53+00:00   USER
Error   PHP Undefined index: MemAvailable at /var/www/html/nextcloud/apps/serverinfo/lib/SystemStatistics.php#85    2016-10-14T05:00:52+00:00   USER
Error   PHP Undefined index: MemAvailable at /var/www/html/nextcloud/apps/serverinfo/lib/SystemStatistics.php#85    2016-10-14T05:00:51+00:00   USER
Error   PHP Undefined index: MemAvailable at /var/www/html/nextcloud/apps/serverinfo/lib/SystemStatistics.php#85    2016-10-14T05:00:50+00:00   USER
Error   PHP Undefined index: MemAvailable at /var/www/html/nextcloud/apps/serverinfo/lib/SystemStatistics.php#85    2016-10-14T05:00:49+00:00   USER
Error   PHP Undefined index: MemAvailable at /var/www/html/nextcloud/apps/serverinfo/lib/SystemStatistics.php#85    2016-10-14T05:00:48+00:00   USER
Error   PHP Undefined index: MemAvailable at /var/www/html/nextcloud/apps/serverinfo/lib/SystemStatistics.php#85    2016-10-14T05:00:47+00:00   USER
Error   PHP Undefined index: MemAvailable at /var/www/html/nextcloud/apps/serverinfo/lib/SystemStatistics.php#85    2016-10-14T05:00:46+00:00   USER
Error   PHP Undefined index: MemAvailable at /var/www/html/nextcloud/apps/serverinfo/lib/SystemStatistics.php#85    2016-10-14T05:00:45+00:00   USER
```
#### Log file (data/owncloud.log)

```
{"reqId":"oQMHwd3XXtIIczPsm8ye","remoteAddr":"IPADRESS","app":"PHP","message":"mkdir(): No such file or directory at \/var\/www\/html\/CLOUDFOLDER\/lib\/private\/Setup.php#289","level":3,"time":"2016-10-10T20:58:31+00:00","method":"POST","url":"\/CLOUDFOLDER\/index.php","user":"--"}
{"reqId":"aC5pWneMfRyb+SP2cGL8","remoteAddr":"IPADRESS","app":"PHP","message":"mkdir(): No such file or directory at \/var\/www\/html\/CLOUDFOLDER\/lib\/private\/Setup.php#289","level":3,"time":"2016-10-10T21:00:11+00:00","method":"POST","url":"\/CLOUDFOLDER\/index.php","user":"--"}
{"reqId":"NEULxRLP2NxZdKXG52ou","remoteAddr":"IPADRESS","app":"mysql.setup","message":"Specific user creation failed: An exception occurred while executing 'SELECT user FROM mysql.user WHERE user=?' with params [\"oc_USER\"]:\n\nSQLSTATE[42000]: Syntax error or access violation: 1142 SELECT command denied to user 'CLOUDFOLDERuser'@'localhost' for table 'user'","level":3,"time":"2016-10-10T21:00:47+00:00","method":"POST","url":"\/CLOUDFOLDER\/index.php","user":"--"}
{"reqId":"NEULxRLP2NxZdKXG52ou","remoteAddr":"IPADRESS","app":"mysql.setup","message":"Database creation failed: An exception occurred while executing 'GRANT ALL PRIVILEGES ON `CLOUDFOLDER` . * TO 'CLOUDFOLDERuser'':\n\nSQLSTATE[42000]: Syntax error or access violation: 1044 Access denied for user 'CLOUDFOLDERuser'@'localhost' to database 'CLOUDFOLDER'","level":3,"time":"2016-10-10T21:00:47+00:00","method":"POST","url":"\/CLOUDFOLDER\/index.php","user":"--"}

```
#### Browser log

```
GET 
XHR 
https://MYDOMAIN/CLOUDFOLDER/ocs/v2.php/apps/notifications/api/v1/notifications [HTTP/1.1 200 OK 214ms]
NS_ERROR_ILLEGAL_VALUE: Component returned failure code: 0x80070057 (NS_ERROR_ILLEGAL_VALUE) [nsIX509Cert.equals]ssl-observatory.js:472
GET 
XHR 
https://MYDOMAIN/CLOUDFOLDER/ocs/v2.php/apps/notifications/api/v1/notifications [HTTP/1.1 200 OK 200ms]
NS_ERROR_ILLEGAL_VALUE: Component returned failure code: 0x80070057 (NS_ERROR_ILLEGAL_VALUE) [nsIX509Cert.equals]
```

Greetz
razorfall

**Comment by JarbyDev** [comment]
I have the Same issue.

**Comment by nickvergessen** [comment]
> PHP Undefined index: MemAvailable

has been fixed with 10.0.2, feel free to report a new issue, if something is still open
